config/locales/en.yml

```diff
@@ -106,10 +106,10 @@ en:
 
     results:
       title: Your Results
-      description: |
+      description_html: |
         Minimum pension contribution rates will increase in April 2018 and again in April 2019.
         These figures show how this will affect your contributions.
-        Contributions will be based on your earnings of %{eligible_salary} a year.
+        Contributions will be based on your earnings of <span data-dough-eligible-salary>£%{eligible_salary}</span> a year
```

Comment:
the eligible salary variable includes the currency symbol so no need to include it in the yml file, otherwise you will get 2 pound signs.

Reply:
Odd, that would make sense though was not reflected in the screenshot above.

Reply:
Nonetheless, it is a problem and has been updated.